### PR TITLE
feat: add department card component

### PR DESCRIPTION
**Pull request by diogogmatos**
image
- **chore: update tailwindcss**
- **feat: add department card component**

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -1,33 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200");
```
Comment (ruilopesm) [inline src/app/globals.css:1]: When requesting a google font, the user's ip address is sent within that request. Unfortunately, this is against the GDPR and we would be violating it. I know we are making this very same thing on other platforms and we should improve it on those too.

My suggestion is to download the font and reference it locally. This way we would be compliant with the GDPR.

Learn more [here](https://www.cookieyes.com/documentation/google-fonts-and-gdpr/).

Besides that, what a great work. Keep up with it 🙌

File: src/components/department_card.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import Link from "next/link";
```
Comment (ruilopesm) [inline src/components/department_card.tsx:1]: In my opinion, a better casing for this file would be kebab-case. Resulting in `department-card`. This is a standard that is starting to get some adoption 👀

Comment (diogogmatos) [reply src/components/department_card.tsx:1]: Agreed.

File: src/app/globals.css (hunk 1 above)
Comment (diogogmatos) [reply src/app/globals.css:1]: Fixed! Thanks for the heads up! 🙏

**Review by joaodiaslobo — APPROVED** [review]
Looks great! 🤩

**Pull request merged**